atm.js

```diff
@@ -1,3 +1,91 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
 });
+
+checkingDeposit = $("#checking").find('.deposit');
+checkingWithdraw = $("#checking").find('.withdraw');
+savingsDeposit = $("#savings").find('.deposit');
+savingsWithdraw = $('#savings').find('.withdraw');
+checkingBalance = 0;
+savingsBalance = 0;
+// function show() {
+//   console.log('hello');
+// }
+// update checking balance with deposit
+function updateCheckingDeposit () {
+  checkingBalance += parseInt($('#checking .input').val());
+  $("#checking .balance").html('$' + checkingBalance);
+  $('#checking .input').val("");
+  redChecking();
+}
+
+
+// update checking balance with withdraw
+function updateCheckingWithdraw () {
+  if (checkingBalance < parseInt($('#checking .input').val())) {
+    alert('You have insufficient funds!');
+    $('#checking .input').val("");
+  }
+  else {
+    checkingBalance -= parseInt($('#checking .input').val());
+    $("#checking .balance").html('$' + checkingBalance);
+    $('#checking .input').val("");
+    redChecking();
+  }
+}
+
+
+
+
+
+// update savings balance with deposit
+function updateSavingsDeposit () {
+  savingsBalance += parseInt($('#savings .input').val());
+  $("#savings .balance").html('$' + savingsBalance);
+  $('#savings .input').val("");
+  redSavings();
+}
+
+
```

Comment:
I like that you've broken the updating out into functions. Very common pattern in development.